src/main/cpp/RobotContainer.cpp

```diff
@@ -1,24 +1,29 @@
-/*----------------------------------------------------------------------------*/
-/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
-/* Open Source Software - may be modified and shared by FRC teams. The code   */
-/* must be accompanied by the FIRST BSD license file in the root directory of */
-/* the project.                                                               */
-/*----------------------------------------------------------------------------*/
-
 #include "RobotContainer.h"
 
-RobotContainer::RobotContainer() : m_autonomousCommand(&m_subsystem) {
+#include "commands/DriveManual.h"
+
+
+RobotContainer::RobotContainer()  /*: m_driveManual(&m_drivetrain)*/ {
   // Initialize all of your commands and subsystems here
 
+  m_drivetrain.SetDefaultCommand(DriveManual(&m_drivetrain,
+      [this] { return mGamepadDriver.GetY(frc::GenericHID::kLeftHand); },
```

Comment:
When using Lambdas or other functions, make sure the logic in the function (aka between the `{` and `}`) are on new lines.

Helps with code readability